Userland/Services/WebDriver/Session.h

```diff
@@ -14,6 +14,7 @@
 #include <LibCore/Promise.h>
 #include <LibWeb/WebDriver/Error.h>
 #include <LibWeb/WebDriver/Response.h>
+#include <WebContent/WebDriverConnection.h>
```

Comment:
This should not be included in the header, it's not needed here

Reply:
I added this because I made `WebContent::WebDriverConnection::get_property` a public function in `WebContent/WebDriverConnection.h` since the window-switching code needs the functionality as well. Is there a better way to do this? I'm not that experienced with C++ best practices.

Reply:
it's only used in the actual implementation, not the header, so should go into the cpp file :)

Reply:
Thanks! I'll fix that